### ISSUE TITLE
Can't keep "Settings" tab and "Release Notes" both open

### ISSUE THREAD

**Issue by glittle** [Microsoft/vscode#17428]
- VSCode Version: Code 1.8.0 (38746938a4ab94f2f57d9e1309c51fd6fb37553d, 2016-12-13T17:45:32.595Z)
- OS Version: Windows_NT ia32 6.3.9600
- Extensions:

|Extension|Author|Version|
|---|---|---|
|Theme-Flat|gerane|0.0.2|
|easy-less|mrcrowl|1.2.5|
|csharp|ms-vscode|1.5.3|
|vs-keybindings|ms-vscode|0.1.1|
|sass-indented|robinbentley|1.2.1|

---

Steps to Reproduce:

1.  View Release Notes
2.  From menu... File / Preferences / Workplace Settings
3.  Release Notes are closed!
4.  From menu... Help / Release Notes
5.  Settings is closed!

If the settings.json file is being edited, it is not closed.

--> The Release Notes should be treated as a standard tab.

**Comment by kieferrm** [comment]
@glittle The tabs are opened in what we call preview mode. If you double click the tab of the settings editor after step 2 you will see that it is still open after step 4. 

Closing in favor of #17431.

**Comment by bpasero** [comment]
There is also a global setting to control this: `"workbench.editor.enablePreview": false`
